colin/checks/best_practices.py

```diff
@@ -70,11 +73,11 @@ def __init__(self):
 
 
 class HelpFileCheck(FileSystemCheck):
+    name = "help_file_required"
```

Comment:
If we don't need to keep some 'backward compatibility' here, then I'd remove the `_required`, so it's just `help_file`.

Reply:
Yes, I hope it is the last one.

It needs to be changed in the rulesets as well, but as I see, I've completely removed that check and overwrite the properties of the `help_file_or_readme`. -- I'll remove that check.

Thanks.